### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_Organizations_Email Notifications_HighContrast: In High contrast mode Keyboard focus is not completely visible on "Check boxes".

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/NuGetGallery#8924]
### Impact

It's more difficult to complete my work

### Describe the bug

Source: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1437973

### Repro Steps

1. Hit the URL: https://www.nuget.org/ 
1. Tab navigate to "Sign in" control and hit enter.
1. Tab navigate to "Sign in with Microsoft" control and hit enter.
1. Tab navigate to “Manage Organizations” from the Profile Dropdown and hit enter.
1. Tab navigate to "Add New" control and hit enter.
1. Enter values in "Organization Name" and "Organization Email Address" and hit enter on "Add" Control.
1. Tab navigate to “Manage Organizations” from the Profile Dropdown and hit enter.
1. "Organizations" screen gets opened.
1. Turn ON High Contrast mode and verify that all the elements of "Email Notifications" and check keyboard focus is completely visible or no "check boxes" in high contrast mode.

In High contrast mode keyboard focus is not completely visible on "check boxes" when we navigate using keyboard.

Note: This issue is also reproducing in select scope check box in API keys.

### Expected Behavior

The Keyboard focus should be completely visible on "Check boxes" when we navigate using keyboard.
